presto-docs/src/main/sphinx/release/release-335.rst

```diff
@@ -0,0 +1,51 @@
+=========================
+Release 335 (XX Jun 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix failure when :func:`reduce_agg` is used as a window function. (:issue:`3883`)
+* Fix incorrect cast from ``TIMESTAMP WITHOUT TIME ZONE`` to ``TIME`` type. (:issue:`3848`)
+* Fix incorrect query results when converting very large ``TIMESTAMP`` values into
```

Comment:
> converting ... `TIMESTAMP` into `TIMESTAMP WITHOUT TIME ZONE`

I don't understand what this is saying, since they are the same type. The optional `WITHOUT TIME ZONE` part is syntactic "noise words" (as the PostgreSQL documentation would say) and we don't ever document or display that. Users know it as the `TIMESTAMP` type.

Reply:
In the doc PR that should be merged we decided to call it by the full name .. I guess thats why the full name is used here. Either way.. we should just link to the docs.

Reply:
And in this specific issue .. I am not sure what this is fixing either .. @martint ?

Reply:
I think that's a typo. It should be "WITH TIME ZONE"

Reply:
confirm & apologies; WITH time zone it ought to be